4. Operators-and-onstructors/Operators-and-constructors-task1.rb

```diff
@@ -0,0 +1,56 @@
+#1. Create method which gets 10 random integers (from 0 to 9) (generate it using while or until)
```

Comment:
first of all we should have a method as task says
and second: try to reduce lines number as much as possible